src/kubernetes-ingress-controller/references/version-compatibility.md

```diff
@@ -58,6 +64,31 @@ other enterprise functionality, built on top of the Open-Source {{site.base_gate
 | Kubernetes 1.24           | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 | Kubernetes 1.25           | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
 
+### Gateway API
+
+The following table presents the compatibility of {{site.kic_product_name}}'s [Gateway API][gateway-api]
+features with specific Kubernetes minor versions. As {{site.kic_product_name}} implements these features on top of the upstream
+project which defines [its own compatibility declarations][gateway-api-supported-versions], the expected compatibility
+of Gateway API features might be limited to those.
+
+| {{site.kic_product_name}} |            2.1.x            |            2.2.x            |            2.3.x            |            2.4.x            |            2.5.x            |            2.6.x            |            2.7.x            |
```

Comment:
```suggestion
| KIC Gateway API |            2.1.x            |            2.2.x            |            2.3.x            |            2.4.x            |            2.5.x            |            2.6.x            |            2.7.x            |
```

Reply:
@czeslavo Is my change accurate? It just felt like this was referring to only KIC Gateway API compatibility with Kubernetes.

Reply:
The whole matrix describes KIC's Gateway API compatibility with Kubernetes versions, but I think that the name used here should be left as `{{site.kic_product_name}}` as it refers to the exact versions of our product.